invadedhouse/run.py

```diff
@@ -21,9 +20,9 @@
 def main():
     cprint('   ____                       __         __  __ __                   ', 'red')
     cprint('  /  _/___   _  __ ___  _ ___/ /___  ___/ / / // /___   __ __ ___  ___', 'yellow')
-    cprint(' _/ / / _ \ | |/ // _  `// _  // -_)/ _  / / _  // _ \ / // /(_-</  -_)', 'green')
-    cprint('/___//_//_/ |___/ \_,_/ \__,_/ __/ \_,_/  /_//_/ ___/ \__,_//___/ \__/ ', 'cyan')
-    print('')
+    cprint(' _/ / / _ \\ | |/ // _  `// _  // -_)/ _  / / _  // _ \\ / // /(_-</  -_)', 'green')
+    cprint('/___//_//_/ |___/ \\_,_/ \\__,_/ __/ \\_,_/  /_//_/ ___/ \\__,_//___/ \\__/ ', 'cyan')
+    print()
```

Comment:
I know these lines look weird in the source code but it fixes bugs in parsing strings

Reply:
why is the logo cut in half

Reply:
It's not... go [this view](https://github.com/redjoey/games/pull/12/files) to see the changes better